app/js/map/map.factory.js

```diff
@@ -58,6 +59,12 @@
               return map.getBounds();
             }
 
+            function createLatLng(latitude, longitude) {
```

Comment:
Is this necessary? GMaps accepts LatLngLiterals (= objects with lat and lng attributes) with no problems.
https://developers.google.com/maps/documentation/javascript/3.exp/reference#LatLngLiteral